Dockerfile.sdk

```diff
@@ -133,6 +133,8 @@ RUN cmake -DCMAKE_INSTALL_PREFIX=/workspace/install \
           -DTRITON_ENABLE_JAVA_HTTP=ON \
           -DTRITON_ENABLE_PERF_ANALYZER=ON \
           -DTRITON_ENABLE_PERF_ANALYZER_C_API=ON \
+          -DTRITON_ENABLE_TF_SERVING=ON \
+          -DTRITON_ENABLE_TORCHSERVE=ON \
```

Comment:
These flags are meant to enable TFS and TS client backends in perf_analyzer. To avoid confusion:
```
TRITON_ENABLE_TF_SERVING -> TRITON_ENABLE_PERF_ANALYZER_TFS
TRITON_ENABLE_TORCHSERVE -> TRITON_ENABLE_PERF_ANALYZER_TS
```
Or something on those lines.

Reply:
We have a ticket to move perf_analyzer to its own repo. All these things should be simplified.

Reply:
Thanks for the review. Flags are updated here and in client repo as well.